docs/_static/niscope_usage.inc

```diff
@@ -10,8 +10,98 @@ The following is a basic example of using the **niscope** module to open a sessi
         session.configure_vertical(range=1.0, coupling=niscope.VerticalCoupling.AC)
         session.configure_horizontal_timing(min_sample_rate=50000000, min_num_pts=1000, ref_position=50.0, num_records=1, enforce_realtime=True)
         with session.initiate():
-            wfm, wfm_infos = session[args.channels].fetch(num_samples=args.length)
-        print('Number of samples acquired: {:,}\n'.format(len(wfm)))
+            waveforms = session[[0,1]].fetch()
+        print('Number of samples acquired: {:,}\n'.format(len(waveforms.wfm)))
+
+The waveform returned from `fetch <http://nimi-python.readthedocs.io/en/master/niscope/functions.html#niscope.Session.fetch>`_ is a 1D list of Python objects
```

Comment:
* I don't think there are 2D lists, so 1D is redundant.
* Instead of Python objects, can you refer to the type and maybe make it so it links to it?

Reply:
I was trying to convey that it is `[0, 1, 2, 3, 4, 5]` rather than `[[0, 1, 2,], [3, 4, 5]]`

How should I say that if not 1D? Flat?